src/main/java/cloud/orbit/web/EmbeddedJettyServer.java

```diff
@@ -126,6 +126,12 @@
     @Config("orbit.jetty.ssl.clientAuth.enabled")
     private boolean clientAuth = false;
 
+    @Config("orbit.jetty.ssl.protocols.include")
+    private List<String> sslIncludedProtocols = new ArrayList<>();
+
+    @Config("orbit.jetty.ssl.protocols.exclude")
+    private List<String> sslExcludedProtocols = new ArrayList<>();
```

Comment:
consider Collections.emptyList() instead of creating a new arraylist if you are not going to add to the list or set as null.